pkg/tmpl/context_funcs.go

```diff
@@ -66,6 +66,8 @@ func (c *Context) Exec(command string, args []interface{}, inputs ...string) (st
 
 	cmd := exec.Command(command, strArgs...)
 	cmd.Dir = c.basePath
+	// inherit the environment variables from the parent process
+	cmd.Env = os.Environ()
```

Comment:
@yxxhero Have you tried it without this change? I thought Go's `exec.Command` implicitly inherited the envvars from the process when `cmd.Env == nil`.

Reply:
I will test it without this change. wait a monment.

Reply:
@mumoshu you are right. Thanks very much.
```yaml
If Env is nil, the new process uses the current process's environment.
```